modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/AbstractEventEndpoint.java

```diff
@@ -1824,6 +1827,55 @@ public Response getPublication(@PathParam("eventId") String eventId, @PathParam(
     return okJson(publicationToJSON(publication));
   }
 
+  @GET
+  @Path("{eventId}/tobira/pages")
+  @RestQuery(
+          name = "getEventHostPages",
+          description = "Returns the pages of a connected Tobira instance that contain the given event",
+          returnDescription = "The Tobira pages that contain the given event",
+          pathParameters = {
+                  @RestParameter(
+                          name = "eventId",
+                          isRequired = true,
+                          description = "The event identifier",
+                          type = STRING
+                  ),
+          },
+          responses = {
+                  @RestResponse(
+                          responseCode = SC_OK,
+                          description = "The Tobira pages containing the given event"
+                  ),
+                  @RestResponse(
+                          responseCode = SC_NOT_FOUND,
+                          description = "Tobira doesn't know about the given event"
+                  ),
+                  @RestResponse(
+                          responseCode = SC_SERVICE_UNAVAILABLE,
+                          description = "Tobira is not configured (correctly)"
+                  ),
+          }
+  )
+  public Response getEventHostPages(@PathParam("eventId") String eventId) {
+    var tobira = TobiraService.getTobira(getSecurityService().getOrganization().getId());
+    if (!tobira.ready()) {
+      return Response.status(Status.SERVICE_UNAVAILABLE)
```

Comment:
When testing I am always running into this response. It seems that `TobiraService.getTobira([...])` creates a new `TobiraService`, but all the fields in the service are `null`. Not sure how this is supposed to work, if there's something I forgot to configure please tell me.

Reply:
Nvm, the issue was missing configuration (which should be better documented in the Tobira docs, so nothing to change here)